recipes/default.rb

```diff
@@ -66,3 +66,22 @@
   end
   action [:create, :start]
 end
+
+magic_shell_environment 'DOCKER_HOST' do
+  value node['osl-docker']['service']['host']
+  only_if { node['osl-docker']['service']['host'] }
+end
+
+cron 'docker_prune_volumes' do
+  minute 15
+  environment(DOCKER_HOST: node['osl-docker']['service']['host']) if node['osl-docker']['service']['host']
+  command '/usr/bin/docker system prune --volumes -f > /dev/null'
+end
+
+cron 'docker_prune_images' do
+  minute 45
+  hour 2
+  weekday 0
+  environment(DOCKER_HOST: node['osl-docker']['service']['host']) if node['osl-docker']['service']['host']
+  command '/usr/bin/docker system prune -a -f > /dev/null'
+end
```

Comment:
Upon thinking about this some more, I'm not sure this cronjob is needed anymore. It will basically remove any unused images that don't have containers currently running for. However, if a developer had their docker compose environment down for whatever reason, they will need to run docker build again.

Not the most horrible thing in the world, but perhaps might be a little too aggressive.

@osuosl-cookbooks/chefs thoughts?

Reply:
It might still be fine to keep this, but putting the interval to something like every week would work better. Images aren't created nearly as often as containers, so it shouldn't build up as much.

Reply:
As this stands, it should only run once per week.

Reply:
didn't realize what weekday meant, whoops!

This should be fine then, I don't see anything wrong with this, +1